uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -99,6 +99,33 @@ public String acquireToken(@NonNull final String username,
         return result.getText();
     }
 
+    // click on button acquire token interactive
+    public void acquireTokenWithoutFetchingToken(@NonNull final String username,
```

Comment:
I have no idea what it means to acquireTokenWithoutFetchingOne. Perhaps the Javadoc once it is added will help